packages/perseus/src/widgets/interactive-graphs/interactive-graph-question-builder.ts

```diff
@@ -324,13 +324,15 @@ class InteractiveGraphQuestionBuilder {
             showPoint2: options?.showPoint2 ?? false,
             color: options?.color ?? "grayH",
             lineStyle: options?.lineStyle ?? "solid",
-            labels: (options?.labels ?? []).map((label) => ({
-                type: "label",
-                coord: label.coord ?? vec.midpoint(point1, point2),
-                text: label.text,
-                color: options?.color ?? "grayH",
-                size: label.size ?? "medium",
-            })),
+            labels: options?.labels
+                ? options.labels.map((label) => ({
+                      type: "label",
+                      coord: label.coord ?? vec.midpoint(point1, point2),
+                      text: label.text,
+                      color: options?.color ?? "grayH",
+                      size: label.size ?? "medium",
+                  }))
+                : undefined,
```

Comment:
We can use optional chaining here again for the same result: 

```suggestion
            labels: options?.labels?.map((label) => ({
                      type: "label",
                      coord: label.coord ?? vec.midpoint(point1, point2),
                      text: label.text,
                      color: options?.color ?? "grayH",
                      size: label.size ?? "medium",
                  })),
```